RubberduckTests/Refactoring/MoveCloserToUsageTests.cs

```diff
@@ -912,6 +912,7 @@ public void MoveCloser_RespectsObjectProperties_InUsages()
             using (var state = MockParser.CreateAndParse(vbe.Object))
             {
                 var messageBox = new Mock<IMessageBox>();
+                messageBox.Setup(mb => mb.Show("Prevents an error during the test"));
```

Comment:
let's see how the merge commit build goes, but if it fails this could be related.

Reply:
FWIW, all the tests passed (including this one) on my local machine before I sync'd

Reply:
@BZngr should be good - wouldn't be the first time AV builds go nuts 😉

Reply:
@Vogel612  had the same situation. AIUI, it's only up in AV that it decided it's in its own imaginary world where everything is on fire and it's *fine*.